src/mscorlib/src/System/Resources/IResourceManager.cs

```diff
@@ -0,0 +1,20 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System;
+using System.Globalization;
+using System.Runtime.CompilerServices;
+
+namespace System.Resources
+{
+    /// <summary>
+    /// Private interface between CoreLib and ResourceManager so that CoreLib can access resources
```

Comment:
This won't be just CoreLib, right? All assemblies that are lower than ResourceManager (e.g. System.Private.Uri) will need to be on the same plan.